ts/components/messages/MessagesArchive.tsx

```diff
@@ -14,7 +14,9 @@ import MessageList from "./MessageList";
 
 const styles = StyleSheet.create({
   listWrapper: {
-    flex: 1
+    flex: 1,
+    borderTopWidth: Platform.OS === "android" ? 0.1 : undefined,
+    elevation: 0.1
```

Comment:
Since this kind of style is used in four places, can we share the same borderTopWidth, declare the style once in a common place and share it throughout the components that need it?